content/blog/2017/2017-01-18-refactoring-conditional-to-pipeline.adoc

```diff
@@ -0,0 +1,475 @@
+---
+layout: post
+title: "Refactoring Conditional BuildSteps into Jenkins Pipeline"
+tags:
+- pipeline
+- freestyle
+- plugins
+- conditional-build-step
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at link:http://cloudbees.com[CloudBees].
+
+== Introduction
+
+With all the new developments in Jenkins Pipeline (and
+link:https://jenkins.io/blog/2017/01/12/declarative-pipeline-beta-2/[Declarative Pipeline on the horizon]),
+it's easy to forget what we did to create "pipelines" before
+*link:https://jenkins.io/doc/book/pipeline/[Pipeline]*.
```

Comment:
@bitwiseman *With all the new developments in Jenkins Pipeline* < First occurrence.